conu/backend/docker/container.py

```diff
@@ -249,28 +249,54 @@ def start(self):
         """
         self.d.start(self.get_id())
 
-    def execute(self, command, exec_create_kwargs=None, exec_start_kwargs=None):
+    def execute(self, command, blocking=True, exec_create_kwargs=None, exec_start_kwargs=None):
         """
-        execute a command in this container -- the container needs to be running
+        Execute a command in this container -- the container needs to be running.
+
+        If the command fails, a ConuException is thrown.
+
+        This is a blocking call by default and writes output of the command to logger
+        using the INFO level -- this behavior can be changed if you set
+        the argument `blocking` to `False`.
+
+        If not blocking, you should consume the returned iterator in order to see logs or know
+        when the command finished:
+
+        ::
+
+            for line in self.execute(["ping", "8.8.8.8"])
```

Comment:
Missing colon and 'blocking=False' argument.

Reply:
nice catch! I fixed even more